liquibase_changelog/patternatlas_componentHostingPatterns.xml

```diff
@@ -0,0 +1,174 @@
+            <column name="icon_url" value="https://raw.githubusercontent.com/PatternAtlas/pattern-atlas-content/addComponentHostingPatterns/icons/Component_Hosting_Patterns/fixed-stack.png"/>
+            <column name="pattern_language_id" value="ddbd6d47-6a71-4e41-8704-8c313f3e819f"/>
+            <column name="paper_ref" value="https://link.springer.com/chapter/10.1007/978-3-031-18304-1_3"/>
+            <column name="deployment_modeling_behavior_pattern" value="false"/>
+            <column name="deployment_modeling_structure_pattern" value="true"/>
+        </insert>
+
+        <insert tableName="pattern">
+            <column name="id" value="ebc49a18-4a96-416c-9dd6-5de89d0d1b89"/>
+            <column name="name" value="Serverful Hosting Pattern"/>
+            <column name="uri" value="https://patternpedia.org/patternLanguages/componentHostingPatterns/serverfulHostingPattern"/>
+            <column name="content" value="{&quot;Icon&quot;: &quot;Enter your input for this section here.&quot;, &quot;Forces&quot;: &quot;Enter your input for this section here.&quot;, &quot;Context&quot;: &quot;Enter your input for this section here.&quot;, &quot;License&quot;: &quot;Yussupov, V.; Soldani, J.; Breitenbücher, U.; Brogi, A. and Leymann, F.  **\&quot;From Serverful to Serverless: A Spectrum of Patterns for Hosting Application Components\&quot;**. In: Proceedings of the 11th International Conference on Cloud Computing and Services Science - CLOSER, pages 268-279. SciTePress, 2021. DOI: [10.5220/0010481002680279](https://www.scitepress.org/PublicationsDetail.aspx?ID=yzuOAgEvhvk=)\n\n[![](https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/by-nc-nd.svg)](https://creativecommons.org/licenses/by-nc-nd/4.0/)&quot;, &quot;Problem&quot;: &quot;How to host a software component while retaining control\nover all deployment stack components and the scaling configuration?&quot;, &quot;Solution&quot;: &quot;Enter your input for this section here.&quot;}"/>
```

Comment:
Apart from the Problem description and the License, most patterns are empty. Is this on purpose or did the export miss something?

Reply:
thanks, added the missing content for all patterns